spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -45,7 +45,9 @@
       visibility: InvenioRdmRecordConverter::OPEN_ACCESS,
       related_url: ["https://doi.org/10.5438/55e5-t5c0"],
       acknowledgments: ["this is an acknowledgement"],
-      abstract: ["this is an abstract"]
+      abstract: ["this is an abstract"],
+      identifier: ["(DOI):46589", "(PMID) 11549777", "cruisemicrobe", "615.778-2", "600-15", "m19090127", "PNB-23-38", \
+                     "HSL.2016.15.0007", "(ISBN 10) 3956501241", "PNB-14-96-a", "PNB-15-10"]
```

Comment:
minor: let's place them one per line for readability (limiting line length between 80 and 90 characters is typically a good size)

Reply:
Agree on one per a line, but I generally prefer ~120 characters. 80 is just too tight IMO

Reply:
Ha! the python PEP8 line size limit is 80 (120 is a 50% increase!), black -a modern formatting tool- puts it at 88 and GitHub seems to display 94-95 char lines. 

Again, I don't mind so much, as long as the result is clean and readable.